doc/cookbook/source/examples/regression/least_squares_regression.rst

```diff
@@ -0,0 +1,53 @@
+========================
+Least Squares Regression
+========================
+
+A Linear regression model can be defined as :math:`y_i = \bf{w}^\top\bf{x_i}` where :math:`y_i` is the predicted value, :math:`\bf{x_i}` is a feature vector and :math:`\bf{w}` is the weight vector. We aim to find the linear function that best explains the data, i.e. minimizes the error or loss function :math:`E(\bf{w})` by finding appropriate :math:`\bf{w}`.
+
+.. math::
+    E({\bf{w}}) = \sum_{i=1}^N(y_i-{\bf w}^\top {\bf x}_i)^2
```

Comment:
this can be removed, just say the squared empirical error. Keep in mind that the API examples are...well API examples. Math should be explained elsewhere